File: Dermatoloska_ordinacija/UI/dermatoloska_mobile/lib/models/recenzija.dart
import 'package:json_annotation/json_annotation.dart';

part 'recenzija.g.dart';

@JsonSerializable()
class Recenzija {
  int? recenzijaId;
  String? sadrzaj;
  DateTime? datum;
  int? proizvodId;
  int? korisnikId;

  Recenzija(this.recenzijaId, this.sadrzaj, this.datum, this.proizvodId, this.korisnikId);

  factory Recenzija.fromJson(Map<String, dynamic> json) => _$RecenzijaFromJson(json);

  Map<String, dynamic> toJson() => _$RecenzijaToJson(this);
}

File: Dermatoloska_ordinacija/DermatoloskaOrdinacija.Model/Recenzija.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DermatoloskaOrdinacija.Model
{
    public class Recenzija
    {
        public int RecenzijaId { get; set; }

        public string? Sadrzaj { get; set; }

        public DateTime? Datum { get; set; }

        public int? ProizvodId { get; set; }

        public int? KorisnikId { get; set; }
    }
}
